.github/workflows/e2e-test-provider-example.yml

```diff
@@ -444,4 +449,7 @@ jobs:
         with:
           projectWriteToken: ${{ secrets.PROJECT_WRITE_TOKEN }}
           test: "terraform-provider-example"
+          refStream: ${{ inputs.ref}}
           provider: ${{ inputs.cloudProvider }}
+          kubernetesVersion: ${{ steps.kubernetes.outputs.version }}
+          clusterCreation: "terraform"
```

Comment:
I'm not setting attestationVariant since the value would have to be parsed from the example tf file. Since we rarely change the default, I think its okay to leave it empty.